sockeye/arguments.py

```diff
@@ -231,6 +231,12 @@ def add_model_parameters(params):
                               help="Enables a context gate which adaptively weighs the decoder input against the"
                                    "source context vector before each update of the decoder hidden state.")
 
+    model_params.add_argument('--layer-normalization', action="store_true",
```

Comment:
if we find a better name, happy to take it. Otherwise I think its not too confusing given the docstring.